packages/docs/docs/docs/deploying.md

```diff
@@ -56,14 +56,20 @@ by now, inside the `my-project` directory you should have a `package.json` file
 a `truffle-config.js` file (created by the `zos` CLI for Truffle), and a
 `zos.json` file (created by `zos` for ZeppelinOS).
 
+Note: if you haven't installed Truffle, run the following command:
+
+```console
+npm install -g truffle
+```
+
 ## Adding a contract
 
 Let's create a very simple contract as an example to be added to the project.
 Name it `MyContract.sol`, and put it in the `contracts` folder with the
 following Solidity code:
 
 ```solidity
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.0;
```

Comment:
Agree on migrating to 0.5, but I'd make sure that we update all code examples in our docs, to avoid having some contracts on 0.4 and others on 0.5. I'm not sure if there are any others around, though.

Reply:
fixed the others